RELEASE-NOTES.txt

```diff
@@ -9,6 +9,8 @@
 - bugfix: fixed UI appearance on cells of Order List when tapping with dark mode enabled.
 - bugfix: Reviews no longer convert to partial dark mode. Dark mode coming soon!
 - bugfix: Order Details now has the right space between cells.
+- Enhancement: Support for dark mode
+- bugfix: Settings no longer convert to partial dark model.
```

Comment:
could probably move these under release 3.2?

Reply:
not sure what "Settings no longer convert to partial dark model" means, does it still apply?